AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java

```diff
@@ -913,18 +913,19 @@ public int cardCount(Long[] ls) {
     /**
      * Bulk delete cards by ID.
      */
-    public void remCards(long[] ids) {
-    	remCards(ids, true);
+    public void remCards(List<Long> ids) {
+        remCards(ids, true);
     }
-    public void remCards(long[] ids, boolean notes) {
-        if (ids.length == 0) {
+
+    public void remCards(java.util.Collection<Long> ids, boolean notes) {
```

Comment:
```suggestion
    public void remCards(Collection<Long> ids, boolean notes) {
```

another fully-qualified that doesn't need to be - if we truly have a clash over the symbol 'Collection' I'd expect something other than java.util.Collection to get the fully-qualified-name treatment :-)